packages/app/src/vis-packs/core/hooks.ts

```diff
@@ -160,3 +162,26 @@ export function useSlicedDimsAndMapping(
     [dimMapping, dims]
   );
 }
+
+export function useIgnoreFillValue(
+  dataset: Dataset
+): ((val: number) => boolean) | undefined {
+  const { attrValuesStore } = useDataContext();
+
+  const rawFillValue = attrValuesStore.getSingle(dataset, '_FillValue');
+  const wrappedFillValue = castArray(rawFillValue);
+
+  const DTypedArray = typedArrayFromDType(dataset.type);
+
+  // Cast fillValue in the type of the dataset values to be able to use `===` for the comparison
+  const fillValue =
+    DTypedArray && typeof wrappedFillValue[0] === 'number'
+      ? new DTypedArray(wrappedFillValue as number[])[0]
```

Comment:
Arf hadn't thought that wrapping as array instead of unwrapping would lead to requiring this type cast. Oh well, it can stay, too bad.